src/native/libs/System.Native/pal_mount.c

```diff
@@ -13,16 +13,24 @@
 #if HAVE_MNTINFO
 #include <sys/mount.h>
 #else
+#if HAVE_SYS_STATFS_H
 #include <sys/statfs.h>
+#endif
 #if HAVE_SYS_MNTENT_H
 #include <sys/mntent.h>
 #include <sys/mnttab.h>
-#include <sys/statvfs.h>
-#else
+#elif HAVE_MNTENT_H
 #include <mntent.h>
 #endif
+#include <sys/statvfs.h>
 #define STRING_BUFFER_SIZE 8192
 
+#ifdef __HAIKU__
```

Comment:
There seems to be uses of both `TARGET_HAIKU` and `__HAIKU__`. Is there a reason we use one versus the other?

Reply:
Not really... I just use the style that is more common in the file.

Reply:
I think since we are introducing Haiku for the first time, it should be consistent.